plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/aps/system/services/mapping/SeoMappingDAO.java

```diff
@@ -79,7 +79,7 @@ public void updateMapping(FriendlyCodeVO vo) {
 		try {
 			conn = this.getConnection();
 			conn.setAutoCommit(false);
-			this.deleteRecord(vo.getFriendlyCode(), vo.getPageCode(), vo.getContentId(), conn);
+			this.deleteRecord(vo.getFriendlyCode(), null, null, conn);
```

Comment:
is this safe? Should the method signature be changed? What about records already in the DB? Do we need a migration script?

Reply:
I've done some other code refactoring and improvement of several methods. Can you check another time?